index.html

```diff
@@ -285,6 +285,23 @@ <h3>Fund Club</h3>
 					<p>Pledge $100/month to fund tech projects by and for marginalized people.</p>
 				</a>
 			</div>
+			<div class="row">
+				<a class="col-md-4" href="https://www.redhat.com/en/about/women-in-open-source" target="_blank" rel="noopener">
+					<img alt="" src="img/projects/womeninopensource.png" />
+					<h3>Women in Open Source Award</h3>
+					<p>We believe open source is the future of technology. It's time to recognize the contributions women are making and inspire a new generation to join the open source movement.</p>
+				</a>
+				<a class="col-md-4" href="https://www.globaldiversitycfpday.com" target="_blank" rel="noopener">
+					<img alt="" src="img/projects/globaldiversitycfpday.jpg" />
+					<h3>Global Diversity CFP Day</h3>
+					<p>Workshops hosted around the globe encouraging and advising newbie speakers to put together your very first talk proposal.</p>
+				</a>
+				<a class="col-md-4" href="https://hacktoberfest.digitalocean.com/" target="_blank" rel="noopener">
+					<img alt="" src="img/projects/hacktober.jpg" />
```

Comment:
is (`alt=""`) alts are empty, why put them in the first place? :) we could remove the attribute altogether.

Reply:
It is to make clear they are decorative images, not just forgotten alt tags. :)

See https://a11yproject.com/posts/alt-text/

> Use an empty alt attribute for any image that is decorative or not necessary for understanding the content of the page. For example, an image providing decoration to an article’s heading could be hidden by giving the image an alt="" or simply alt